diff --git a/node_modules/react-native-countdown-component/index.js b/node_modules/react-native-countdown-component/index.js
index b546b82..00b1892 100644
--- a/node_modules/react-native-countdown-component/index.js
+++ b/node_modules/react-native-countdown-component/index.js
@@ -45,18 +45,21 @@ class CountDown extends React.Component {
     wentBackgroundAt: null,
   };
 
+  changeListener = null
   constructor(props) {
     super(props);
     this.timer = setInterval(this.updateTimer, 1000);
   }
 
   componentDidMount() {
-    AppState.addEventListener('change', this._handleAppStateChange);
+  this.changeListener = AppState.addEventListener('change', this._handleAppStateChange);
   }
 
   componentWillUnmount() {
     clearInterval(this.timer);
-    AppState.removeEventListener('change', this._handleAppStateChange);
+   if (this.changeListener) {
+     this.changeListener.remove()
+    }
   }
 
   componentDidUpdate(prevProps, prevState) {
